CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## MASTER
 * Improve Travis build time
 * Add hostname in /etc/hosts file
+* Add default value to direction parameter when enable ufw to prevent crash in ansible versions upper 2.7.7
```

Comment:
I would add details on the default parameter here so people don't have to go to the code to find it out. Something like
```
Add "incoming" as default value for the "direction" parameter when enabling ufw, to work around issue #53854 (https://github.com/ansible/ansible/issues/53854) in Ansible 2.7.7+
```

Reply:
@javcasas , added in f94126554d93e94d7df42bb3f447b8f7c07ede84